pymc/distributions/distribution.py

```diff
@@ -354,11 +355,268 @@ def dist(
         return rv_out
 
 
+class SymbolicDistribution:
```

Comment:
Will those show up in the docs?

Reply:
We also didn't mention it in the release notes.

Reply:
They show in the docs (you can check the RTD job in this PR). I didn't see the SymbolicDist as being user facing, that's why it's not mentioned in the release notes.

Reply:
I agree, but it's such an important new addition that I think it deserves some more attention.